apps/frontend-pwa/src/pages/editProfile.tsx

```diff
@@ -145,85 +164,179 @@ function EditProfile() {
+                              {values.isProfilePublic
+                                ? t('shared.generic.yes')
+                                : t('shared.generic.no')}
+                            </div>
+                            <div className="flex-1">
+                              <Prose className={{ root: 'prose-sm' }}>
+                                {t('pwa.profile.isProfilePublic')}
+                              </Prose>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
 
-              <Form>
-                <div className="flex flex-col w-full gap-8 mt-8 md:flex-row md:gap-16">
-                  <div className="flex flex-col justify-between flex-1 order-2 gap-4 md:order-1">
-                    {!data.self?.avatar && (
-                      <Prose>{t('pwa.profile.welcomeMessage')}</Prose>
-                    )}
+                    <Button
+                      fluid
+                      type="submit"
+                      disabled={isSubmitting || !isValid}
+                    >
+                      <Button.Icon>
+                        <FontAwesomeIcon icon={faSave} />
+                      </Button.Icon>
+                      <Button.Label>{t('shared.generic.save')}</Button.Label>
+                    </Button>
+                  </div>
 
-                    <div className="space-y-4">
-                      <FormikTextField
-                        name="username"
-                        label={t('shared.generic.username')}
-                        labelType="small"
-                        className={{ label: 'font-bold text-md text-black' }}
-                      />
-                      <FormikTextField
-                        name="password"
-                        label={t('shared.generic.password')}
-                        labelType="small"
-                        className={{ label: 'font-bold text-md text-black' }}
-                        type="password"
-                      />
-                      <FormikTextField
-                        name="passwordRepetition"
-                        label={t('shared.generic.passwordRepetition')}
-                        labelType="small"
-                        className={{ label: 'font-bold text-md text-black' }}
-                        type="password"
-                      />
-                    </div>
+                  <div className="flex-1 order-1 md:order-2 md:bg-slate-50 md:p-4 rounded justify-between flex flex-col space-y-4">
+                    {!data.self?.isActive && (
+                      <div className="flex-initial space-y-2">
+                        <H3 className={{ root: 'border-b mb-0' }}>Privacy</H3>
+                        <Markdown
+                          withProse
+                          withLinkButtons={false}
+                          content={t('pwa.profile.privacyNotice')}
```

Comment:
still being written, will be added to de.json and en.json once done, untranslated string for now